pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -124,6 +125,8 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	operationGenerator OperationGenerator) OperationExecutor {
 
+	util.RegisterMetrics()
```

Comment:
the `sync.once` thing will keep us safe https://github.com/kubernetes/kubernetes/pull/50036/files#diff-7c5ec7e6067fba16973d1249a87653a8R42